OpenRA.Mods.Common/Traits/Health.cs

```diff
@@ -19,9 +19,13 @@ public class HealthInfo : ITraitInfo, UsesInit<HealthInit>
 	{
 		[Desc("HitPoints")]
 		public readonly int HP = 0;
+
 		[Desc("Trigger interfaces such as AnnounceOnKill?")]
 		public readonly bool NotifyAppliedDamage = true;
 
+		[Desc("Distance above CenterPosition for targeting by weapons.")]
```

Comment:
I don't think this makes much sense (doesn't "scale").
Consider a east-to-west facing ship.
Would this make more sense as a worldly vector offset? (consider turret offset)